airflow/dag_processing/manager.py

```diff
@@ -239,13 +239,13 @@ def _run_processor_manager(
         airflow.settings.initialize()
         del os.environ['CONFIG_PROCESSOR_MANAGER_LOGGER']
         processor_manager = DagFileProcessorManager(
-            dag_directory,
-            max_runs,
-            processor_timeout,
-            signal_conn,
-            dag_ids,
-            pickle_dags,
-            async_mode,
+            dag_directory=dag_directory,
+            max_runs=max_runs,
+            processor_timeout=processor_timeout,
+            signal_conn=signal_conn,
+            dag_ids=dag_ids,
+            pickle_dags=pickle_dags,
```

Comment:
```suggestion
            dag_ids=dag_ids,
            pickle_dags=pickle_dags,
            signal_conn=signal_conn,
```

Might as well reorder here as well, no?